doc/user/content/get-started/isolation-level.md

```diff
@@ -80,7 +80,17 @@ large propagation delays. For example, if SQL-transaction T1 happens before SQL-
 table t, and T2 queries materialized view mv where mv is an expensive materialized view including t, then T2 may not see
 all the rows that were seen by T1 if they are executed close enough together in real time.
 
-If a consistent snapshot is not available across all objects in a query, the query will be blocked until one becomes available. On the other hand, if a consistent snapshot is available, the query will be executed immediately. A consistent snapshot is guaranteed to be available for queries that involve a single object (which includes queries against a single materialized view that was created using multiple objects). Such queries will therefore never block, and always be executed immediately.
+If a consistent snapshot is not available across all objects in a query and all other objects in
+the current transaction, then the query will be blocked until one becomes available. On the other
+hand, if a consistent snapshot is available, then the query will be executed immediately. A
+consistent snapshot is guaranteed to be available for transactions that are known ahead of time to
+involve a single object (which includes transactions against a single materialized view that was
+created using multiple objects). Such transactions will therefore never block, and always be
+executed immediately. A transaction can only be known ahead of time to involve a single object when
+using auto-commit (i.e. omitting `BEGIN` and `COMMIT`). When using explicit transactions (i.e.
+starting a transaction with `BEGIN`), then it is assumed that all objects that share a schema with
+any object mentioned in the first query of the transaction may be used later in the transaction.
```

Comment:
This is only true for `SELECT`, not `SUBSCRIBE`. Should the text specify that? It's a bit more separated in the BEGIN docs.

Reply:
Yeah we should specify that, I forgot about that. That's because we only allow a single subscribe query in a transaction right?

Reply:
I just pushed some clarifications.